apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/Chat.tsx

```diff
@@ -0,0 +1,321 @@
+import { useCallback, useContext, useEffect, useRef, useState } from 'react'
+
+import {
+  Chain,
+  CompileError,
+  ContentType,
+  Conversation,
+  Message as ConversationMessage,
+  ConversationMetadata,
+  MessageRole,
+} from '@latitude-data/compiler'
+import {
+  ChainEventTypes,
+  LATITUDE_EVENT,
+  PROVIDER_EVENT,
+  type ChainEvent,
+  type DocumentVersion,
+} from '@latitude-data/core/browser'
+import { Text } from '$ui/ds/atoms'
```

Comment:
⚠️ Wrong import. I'll be your linter until we figure out why auto imports don't pick the package name